phpBB/includes/acp/acp_attachments.php

```diff
@@ -630,6 +630,7 @@ function main($id, $mode)
 				$cat_lang = array(
 					ATTACHMENT_CATEGORY_NONE		=> $user->lang['NO_FILE_CAT'],
 					ATTACHMENT_CATEGORY_IMAGE		=> $user->lang['CAT_IMAGES'],
+					ATTACHMENT_CATEGORY_AUDIO		=> $user->lang['CAT_AUDIO_FILES'],
```

Comment:
You should use language component better, and also don't access the property directly

Reply:
Change in whole file or only in this line.
In ACP language component used only in acp_extensions.php all other files uses user->lang

Reply:
Uhm, only that line

Reply:
You mean using `$user->lang('CAT_AUDIO_FILES')` only because using $language require adding to global, right?

Reply:
Using it as an array is deprecated and Ruben has a task to replace all instances of the old with the new. Adding more of the old deprecated method will not be allowed.